python/lsst/ts/hvac/hvac_csc.py

```diff
@@ -322,51 +393,76 @@ def connected(self):
     def get_config_pkg():
         return "ts_config_ocs"
 
-    def _add_sal_commands(self):
-        # This adds the do_foo_enable commands.
-        for topic in HvacTopic:
-            if topic.value not in xml.TOPICS_ALWAYS_ENABLED:
-                method_name = f"do_{topic.name}_enable"
-                method = functools.partial(self._do_enable, topic=topic)
-                setattr(self, method_name, method)
+    def _add_config_commands(self):
+        # Find all device groups that can be commanded.
+        command_groups = set(
+            k
+            for k, v in DEVICE_GROUPS.items()
+            for i in v
+            if i not in TOPICS_WITHOUT_CONFIGURATION
+        )
+        # This adds the do_configFoos functions.
+        for command_group in command_groups:
+            function_name = f"do_config{to_camel_case(command_group)}s"
+            function = functools.partial(self._do_config)
```

Comment:
Is functools.partial actually doing anything? I see no arguments being added.

I think you can skip this line and use `setattr(self, function_name, self._do_config)`

Reply:
Thanks, done.